.github/workflows/ci.yml

```diff
@@ -95,6 +95,72 @@ jobs:
       - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v8.19.1
       - uses: Alfresco/ya-pmd-scan@v4.3.0
 
+  code_coverage_report:
```

Comment:
Please can we merge this with an existing workflow? Otherwise I think we're running all the unit/integration tests twice.